code/flexbox assignment/style.css

```diff
@@ -0,0 +1,74 @@
+.challenge-1 {
+    display:flex
+}
+
+.challenge-2 {
+    display:flex;
+    flex-direction: row-reverse;
+}
+
+.challenge-3 {
+    display:flex;
+    justify-content:flex-end;
+}
+
+.challenge-4 {
+    display: flex;
+    justify-content: space-around;
+}
+
+.challenge-5 {
+    display:flex;
+    justify-content: center;
+    column-gap: 20px;
+}
+
+/* 
+Tip: This is called the "child selector". 
+It selects all .bubble elements that are direct
+children of .challenge-6 elements. 
+*/
+.challenge-6 .bubble {
+    display:flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.challenge-7 {
+    display: flex;
+}
+
+.challenge-7 .red {
+    flex-grow: 1;
+}
+
+.challenge-8 {
```

Comment:
The instructions were also to center these! See if you can figure it out, but you don't need to turn it again.